SpatialGDK/Source/SpatialGDK/Private/Interop/ActorSystem.cpp

```diff
@@ -2409,6 +2418,7 @@ void ActorSystem::OnEntityCreated(const Worker_CreateEntityResponseOp& Op, FSpat
 
 	AActor* Actor = Channel.Actor;
 	const Worker_EntityId EntityId = Channel.GetEntityId();
+	ensure(EntityId != SpatialConstants::INVALID_ENTITY_ID);
```

Comment:
New ensure based on Mike's comment about how the entity pool works where we should always be creating entities for actors with known entity ids.